### PR TITLE
t3 and previous test updates

### PR DESCRIPTION
**Pull request by JelenaCernysova**

### CODE REVIEW

File: cypress/integration/t-3.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const loginPage = new LoginPage();
+const tasksPage = new TasksPage();
+const createTaskPage = new CreateTaskPage();
+
+describe('Create task', function() {
+
+    beforeEach(function(){
+        cy.loginAs("Admin")
+        cy.visit('/')
```
Comment (DavidV44) [inline cypress/integration/t-3.js]: you could go to /tasks directly, in that case it would be less loading times, because now you go to home page and only then to /tasks.

File: cypress/integration/login.js [hunk 2]
```diff
@@ -22,11 +20,11 @@ describe('Sourcebooks login', function() {
         loginPage.loginWithUserAndRole("Jelena Černyšova", "Team Lead")
 
         cy.url().should('include', '/time-logging');
-        pageComponents.getPageTitle().contains('Timesheets')
+        timeLoggingPage.getPageTitle().contains('Timesheets')
         timeLoggingPage.getCalendar().should('be.visible')
         timeLoggingPage.getTileForm().should('be.visible')
-        pageComponents.getLoggedInUser().contains('Jelena Cernyšova');
-        pageComponents.getNavigationTabsList().should('have.length', 2)
+        timeLoggingPage.getLoggedInUser().contains('Jelena Cernyšova');
```
Comment (DavidV44) [inline cypress/integration/login.js:26]: Why did you change back to using `timeLoggingPage` ? Using `pageComponents` seemed better

**Review by DavidV44 — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js (hunk 2 above)
Comment (JelenaCernysova) [reply cypress/integration/login.js:26]: it uses pageComponents, as TimeLoggingPage class extends pageComponents  class, therefore the logic of pageComponents is in TimeLoggingPage  class without creating additional variables in a test and the logic can still be used

Comment (DavidV44) [reply cypress/integration/login.js:26]: Oh, didn't see that, all good then :)

**Review by DavidV44 — APPROVED** [review]
